<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxShowOnlyDifferent">
        <property name="text">
         <string>Only show different</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTagFilter">
        <property name="text">
         <string>Tag Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTagFilter"/>
      </item>
      <item>
       <widget class="QLabel" name="labelDescFilter">
        <property name="text">
         <string>Description Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDescFilter"/>
      </item>
      <item>
       <widget class="QLabel" name="labelValFilter">
        <property name="text">
         <string>Value filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditValFilter"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelPath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DroppableTreeView" name="treeView">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelPath_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DroppableTreeView" name="treeView_2">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDiff"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionExpand_all"/>
    <addaction name="actionCollapse_all"/>
    <addaction name="actionText_diff"/>
    <addaction name="actionHTML_diff"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="text">
    <string>Diff</string>
   </property>
  </action>
  <action name="actionExpand_all">
   <property name="text">
    <string>Expand all</string>
   </property>
  </action>
  <action name="actionCollapse_all">
   <property name="text">
    <string>Collapse all</string>
   </property>
  </action>
  <action name="actionText_diff">
   <property name="text">
    <string>Text diff</string>
   </property>
   <property name="toolTip">
    <string>Text diff</string>
   </property>
  </action>
  <action name="actionHTML_diff">
   <property name="text">
    <string>HTML diff</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DroppableTreeView</class>
   <extends>QTreeView</extends>
   <header>QDICOMDiffer</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
